notebooks/official/experiments/comparing_local_trained_models.ipynb

```diff
@@ -32,51 +32,48 @@
         "<table align=\"left\">\n",
```

Comment:
<p>we'd like to introduce the following concepts: -&gt; the following concept are explained</p><p> </p><p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>done</p>